src/common/test/run.sh

```diff
@@ -31,6 +31,25 @@ exit_if_fail() {
     fi
 }
 
+if [ -n ${TEST_OPTS} ]; then
```

Comment:
I‘m planning to remove the `run.sh` scripts for unit tests in a long term, this change may be a small block for that.
How about define the default value as
```
DSN_DEFINE_int32(replication,
                 disk_min_available_space_ratio,
#ifdef PEGASUS_UNIT_TEST
                 5,
#else
                 10,
#endif
                 "if disk available space ratio "
```